src/main/java/com/zontik/groshiky/exception/MissionAccountException.java

```diff
@@ -0,0 +1,14 @@
+package com.zontik.groshiky.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class MissionAccountException extends RuntimeException {
```

Comment:
MissedAccountException ?
I'd say that NotFoundAccountException would be more clear way to name it